v5/Dockerfile

```
@@ -1,201 +1,109 @@
+###############################################################################
+# 2) Build (compile / install) stage                                           #
+###############################################################################
+FROM node:${NODE_VERSION} AS builder
+
+# re-declare args inside the stage
+ARG  VERSION
+ARG  GHOST_CLI_VERSION
+ARG  USER
+
+# ---- system packages --------------------------------------------------------
+RUN apk update && apk upgrade --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)